test/ja_serializer/builder/included_test.exs

```diff
@@ -306,13 +306,13 @@ defmodule JaSerializer.Builder.IncludedTest do
     assert "p1" in ids
 
     Application.put_env(:ja_serializer, :key_format, :underscored)
-    json = JaSerializer.format(ArticleSerializer, a1, %{}, include: "author.publishing-agent")
+    json = JaSerializer.format(ArticleSerializer, a1, %{}, include: "author.publishing_agent")
     ids = Enum.map(json["included"], &(Map.get(&1, "id")))
-    assert not "p1" in ids
```

Comment:
Why is this assertion changing?

Reply:
That is because I also changed the string `publishing-agent` to `publishing_agent` two lines above, in order to make a positive assertion to prove that the `:underscored` key format also works.

The same goes for the assertion a few lines down as well.

It might make sense to split this test into multiple different test cases in a `describe "multi-word relationship path keys"` block, and then also have both positive and negative assertions for both `:dasherized` and `:underscored` key formats.